lib/postgrex/protocol.ex

```diff
@@ -1593,6 +1604,16 @@ defmodule Postgrex.Protocol do
     transaction_error(s, postgres)
   end
 
+  defp parse_describe_comment_msgs(query, comment, tail) when is_binary(comment) do
+    statement = query.statement <> "/* #{comment} */"
```

Comment:
Are the comments meant to come before or after? Is there any general feedback here?

Reply:
For general comments it would be better to have it in the front but for sqlcommenter tags the [specification](https://google.github.io/sqlcommenter/spec/) says that it should be at the end.
```
Format

The final comment SHOULD be affixed to the final SQL statement in the format

<SQL STATEMENT> /*<ATTRIBUTE_KEY_VALUE_PAIRS>*/
```
In the examples there is also no spaces around the attributes, I will remove it from my pr too - it's possible to add it easily when you need it but no way to remove it from the caller.

Reply:
As this is part of [opentelemetry](https://cloud.google.com/blog/products/databases/sqlcommenter-merges-with-opentelemetry) maybe we should call it tracing_comment, It would be clear why the query caching is disabled for that.

Reply:
I think as long as we add it to the docs, we should be fine.

Reply:
In this case I just updated the docs. I hope it's in correct place.